WordPress/twentytwentyfive
# Try: Reduce image size.

**Description**

Trying to reduce the image size for several patterns so that the theme zip is lighter.

**Screenshots**

N/A

**Testing Instructions**

404 page
1. Visit the theme 404 page, confirm that the image has good quality.

Patterns:

1. Create a page
2. Add the patterns: Poster-like section, "Overlapping image on the right", "Banner with description and images grid", "About: Image on the right, heading and paragraph", "Events: 3 columns with event images and title", "Link in Bio: Image and heading with tight margins"
3. Confirm that the images maintain good quality.
